modules/ipi-install-troubleshooting-cleaning-up-previous-installations.adoc

```diff
@@ -10,14 +10,14 @@ In the event of a previous failed deployment, remove the artifacts from the fail
 
 . Power off all bare metal nodes prior to installing the {product-title} cluster:
 +
-[source,bash]
+[source,terminal]
 ----
 [kni@provisioner ~]$ ipmitool -I lanplus -U <user> -P <password> -H <management-server-ip> power off
 ----
 
 . Remove all old bootstrap resources if any are left over from a previous deployment attempt:
 +
-[source,bash]
+[source,terminal]
```

Comment:
I'd say this actually is `bash`. There's no `$` and it's a script. The lead text doesn't describe it as such though.